CoffeeChat/Cells/SchedulingPlaceCollectionViewCell.swift

```diff
@@ -41,13 +41,13 @@ class SchedulingPlaceCollectionViewCell: UICollectionViewCell {
     private func setupConstraints() {
         titleLabel.snp.makeConstraints { make in
             make.centerY.equalToSuperview()
-            make.leading.equalToSuperview().inset(12)
-            make.trailing.equalToSuperview().inset(6)
+            make.leading.trailing.equalToSuperview().inset(10)
         }
     }
 
-    func configure(with title: String) {
+    func configure(with title: String, isOtherSide: Bool) {
```

Comment:
In general, can we rename `isOtherSide` to like `isConfirmingTimes` or something like that?